### ISSUE TITLE
[Question] why deprecation notice?

### ISSUE THREAD

**Issue by DmitryEfimenko** [tutumcloud/mongodb#46]
Just like title suggests, why is this library being deprecated?
Official image does not have a way of setting user/password that I'm aware of.

**Comment by richersoon** [comment]
Base on queue on pull request, looks like they are no longer nterested in maintaining this project.

**Comment by gsaslis** [comment]
I have to agree that this repo fills a need that the official docker image doesn't support. 

Setting a username & password ! 

And with all the turmoil around password-less mongos everywhere, I think there's even more of a case to keep this around... 

Just my 0.02
